tests/lib/icmsg/src/main.c

```diff
@@ -11,7 +11,7 @@
  * 212 - sizeof(struct icmsg_buf) - 1 = 199.
  * -1 because internal rd/wr_idx is reserved to mean the buffer is empty.
  */
-static uint8_t memory_area[212];
+static uint8_t memory_area[212] __aligned(4);
```

Comment:
The commit log is just describing the changes without providing any justification for them. Why?

Reply:
Thanks for the updated commit log. I'm going to drop my NAK.